Jellyfin.Plugin.Themerr/ThemerrManager.cs

```diff
@@ -1,4 +1,4 @@
-using System;
+﻿using System;
```

Comment:
What has happened here? The BOM changed on the file?

Reply:
Huh, never run into that before but yeah, it looks like VS removed the BOM. do you want me to change that?

Reply:
Probably best to revert that for now.